app/components/web-components/light-table/vl-group-column/template.hbs

```diff
@@ -3,6 +3,6 @@
   searchField="name"
   propertyToShow="name"
   sortField="name"
-  selectedItems=(await group)
+  selectedItems=(await user.group)
   selectModel=(action "chooseAccountGroup")
```

Comment:
Don't get this change, please explain why this is here

Reply:
Dit hadden we nodig om de group door te geven naar de volgende pagina.